sstables/sstables.cc

```diff
@@ -2471,26 +2472,29 @@ input_stream<char> sstable::data_stream(uint64_t pos, size_t len,
         f = tracing::make_traced_file(std::move(f), std::move(trace_state), format("{}:", get_filename()));
     }
 
+    std::optional<uint32_t> digest = std::nullopt;
```

Comment:
nit: `= std::nullopt` not needed, default constructed optional also contains `nullopt`.